e2e_testing/xfail_sets.py

```diff
@@ -454,7 +453,8 @@
     "_LogSoftmaxModuleStable_basic",
     "LiftFreshCopyModule_basic",
     "ReduceSumDimIntListKeepDimNegativeDimStaticModule_basic",
-    "BroadcastToIdentityCaseStaticModule_basic",
+    "BroadcastToSameRankStaticModule_basic",
```

Comment:
I'm curious - why is the MHLO pass set impacted by a TOSA change ? Is it related to something changed in Utils ?

Reply:
It's because I added a new test which passes.